src/Functions/Public/reservation-service/Remove-vRAReservationStorage.ps1

```diff
@@ -0,0 +1,106 @@
+﻿function Remove-vRAReservationStorage {
+<#
+    .SYNOPSIS
+    Remove vRA reservation storage
+
+    .DESCRIPTION
+    Remove vRA reservation storage
+
+    .PARAMETER Id
+    The Id of the reservation
+
+    .PARAMETER Path
+    The storage path
+
+    .INPUTS
+    System.String.
+
+    .OUTPUTS
+    System.Management.Automation.PSObject
+
+    .EXAMPLE
+    Get-vRAReservation -Name "Reservation01" | Remove-vRAReservationStorage -Path "Datastore01"
+
+#>
+[CmdletBinding(SupportsShouldProcess,ConfirmImpact="High")][OutputType('System.Management.Automation.PSObject')]
```

Comment:
Nothing gets output from this function so can you remove:

[OutputType('System.Management.Automation.PSObject')]